cmake/Modules/SYCL.cmake

```diff
@@ -90,7 +90,7 @@ elseif(is_dpcpp)
   set(CMAKE_CXX_STANDARD 17)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D__SYCL_DISABLE_NAMESPACE_INLINE__=ON -O3 -Xclang -cl-mad-enable")
   if(${BACKEND_DEVICE} STREQUAL "DEFAULT_CPU") 
-    set(DPCPP_SYCL_TARGET spir64_x86_64-unknown-unknown)
+    set(DPCPP_SYCL_TARGET spir64)
```

Comment:
Why does this need changing? The original looks valid from the docs. And why not `spir64_x86_64`?

Reply:
`spir64_x86_64-unknown-unknown` generates code AOT which makes the CI and any other compilations for developing and debugging very slow. Hence, made the default a non-AOT one and added a specific target for Intel CPU which does AOT. 

If people don't like this option, another one would be to accept target triple from the user and remove this elif ladder. Happy to change to that as well. Let me know your thoughts.

Reply:
The proper way is to allow user to pass target-tripple. the `-DTARGET` name should be removed and target triple should allow choosing the backend.

Reply:
Is `DEFAULT_CPU` still a good name for this target?